riva/clients/asr/riva_streaming_asr_client.cc

```diff
@@ -76,6 +76,11 @@ DEFINE_bool(
     "Whether to use SSL credentials or not. If ssl_cert is specified, "
     "this is assumed to be true");
 DEFINE_string(metadata, "", "Comma separated key-value pair(s) of metadata to be sent to server");
+DEFINE_int32(start_history,0, "Value to detect and initiate start of speech utterance");
+DEFINE_double(start_threshold,0., "Threshold value to determine at what percentage start of speech is initiated");
+DEFINE_int32(stop_history,0, "Value to detect endpoint and reset decoder");
+DEFINE_int32(stop_history_eou,0, "Value to detect endpoint and generate an intermediate final transcript");
+DEFINE_double(stop_threshold,0., "Threshold value to determine when endpoint detected");
```

Comment:
similar comment here about the default values, shouldn't we pick them from https://gitlab-master.nvidia.com/dl/riva/riva-speech/-/blob/main/python/servicemaker/servicemaker/modules/asr.py#L558-592 to avoid setting 0 if nothing is specified

Reply:
I have pushed new changes, where default value is -1 and a check if done from both server and client side to ensure these values passed by clients are positive. Please Review

Reply:
LGTM